controllers/PageController.js

```diff
@@ -18,7 +18,7 @@ const index = async (req, res) => {
         if(result){
             songs = result;
         }
-    });
+    }).clone();
```

Comment:
klonujesz metodę i to co zwraca, żeby uniknąć ponowienia zapytania (z tymi samymi parametrami, które zwróci ten sam wynik) do bazy danych i uniknąć niepotrzebnego przeciążenia, dobrze rozumiem?

Reply:
Clone jest potrzebny przy nowej wersji biblioteki mongoose, bo nie wysyła rządania, bo takie już było niedawno wykonane.